composer.json

```
@@ -3,7 +3,7 @@
     "description": "Demo Content migrations for Y Layout Builder module.",
     "type": "drupal-module",
     "require": {
-        "ycloudyusa/y_lb" : "^3",
+        "ycloudyusa/y_lb" : "^3.3.0",
```

```suggestion
        "ycloudyusa/y_lb" : "^3.3.0",
        "drupal/ws_event": "^1.4.3"
```

Should we add this? Ot y_lb is covering needed smart_date modules availability? @froboy

---

Y_lb 3.3.0 covers this dependency. If we need ws_event here then we need dependencies from all other migrations too. :)